tiatoolbox/models/architecture/hovernet.py

```diff
@@ -473,6 +474,8 @@ def _proc_np_hv(np_map: np.ndarray, hv_map: np.ndarray):
             hv_map (np.ndarray): An array of shape (heigh, width, 2) which
               contains the horizontal (channel 0) and vertical (channel 1)
               of possible instances exist withint the images.
+            fx (float): The microns per pixel resolution of the input images. Default
+              is 0.5 for HoVer-Net.
```

Comment:
This is not micron per pixel, Better keep it simply as scale factor to resize kenel size or region of interests. And mention the assume size is at 0.25mpp.

Reply:
Edited to address this.